partner_title_active/tests/test_partner_title.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPartnerTitle(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Partner = cls.env["res.partner"]
+
+    def test_01_partner_title(self):
+        partner = self.Partner.create({"name": "A Partner"})
+        self.assertTrue(partner.active, "Partner should be active by default")
+        partner.action_archive()
+        self.assertFalse(partner.active, "Partner should be inactive after archiving")
```

Comment:
This doesn't test whether the `title` is active or not :confused:

Reply:
Why? Can you describe pls a test scenario you think is more suitable?

Reply:
The partner that is created doesn't have a **title** set.
The scenario might be: 
1. create a partner with a title (res.partner.title)
2. test the title is active
3. archive the title, assert it is not active

Reply:
ok, I'll adapt, thanks

Reply:
ahaha @vvrossem , I've just get that I was archiving the partner and not the title :smiling_face_with_tear: 
good catch, shame on me!